rules-topics/01_introduction.md

```diff
@@ -90,30 +90,27 @@ Status that require Identify:
 |Spirit Bottle|
 
 ### Floating Reset
-Some skills or abilities are listed as "Daily".  These skills and abilities are renewed between 4:00 PM and 6:00 PM.  If you are on a module between that time frame, you must attempt the reset immediately after the mod.  Daily skills and abilities require 10 minutes of concentration/rest to refresh.
+Some skills or abilities are listed as "Daily".  These skills and abilities are renewed between 4:00 PM and 6:00 PM.  If a player is on a module (an event during game run by plot) between that time frame, they must attempt the reset immediately after it ends.  Daily skills and abilities require 10 minutes of concentration/rest to refresh.
 
 ### How Does Combat Work
-
-Chronicles is a boffer combat LARP that focuses as much on roleplay and interaction between PCs and NPCs as physical combat.  The goal of combat is for all participants and observers to be safe.  We emphasize light combat.  A general rule is, if your swing is enough to hurt, someone, you are probably swinging too hard.  While accidents may happen, you are more likely to twist an ankle on uneven ground than hurt yourself during combat.  More details can be found in the Combat section of the rulebook.
+Chronicles is a boffer combat LARP that focuses as much on roleplay and interaction between PCs and NPCs as physical combat.  The goal of combat is for all participants and observers to be safe.  We emphasize light combat.  A general rule is, if a player's swing is enough to hurt, someone, they are probably swinging too hard.  While accidents may happen, someone is more likely to twist an ankle on uneven ground than hurt themselves during combat.  More details can be found in the Combat section of the rulebook.
 
 ### Plot Members and Marshals
+Plot members and marshals are officials of Chronicles.  They will give rulings, run a mod, observe a ritual, and other out of game (OOG) tasks that are needed throughout a market faire.  If a player has a question on a rule or how something works, they will need to ask the appropriate marshal.  
```

Comment:
Plot members and marshals are part of the staff of Chronicles. The Plot Team serves the same function as a Game Master does in a table top RPG. 

Should also add Logistics here.

Reply:
Added.